src/coinbase/address/wallet_address.ts

```diff
@@ -74,6 +74,17 @@ export class WalletAddress extends Address {
     this.key = key;
   }
 
+  /**
+   * Gets the private key.
```

Comment:
make comments similar to https://github.com/coinbase/coinbase-sdk-ruby/blob/master/lib/coinbase/address/wallet_address.rb#L158